enginetest/queries/json_scripts.go

```diff
@@ -17,6 +17,18 @@ package queries
 import "github.com/dolthub/go-mysql-server/sql"
 
 var JsonScripts = []ScriptTest{
+	{
```

Comment:
maybe add a test here of a JSON value that's greater than 65K, but less than the limit